legi/sql/schema.sql

```diff
@@ -27,8 +27,8 @@ CREATE TABLE textes_versions
 ( id char(20) unique not null
 , nature text
 , titre text
-, titrefull text
-, titrefull_s text
+, titrefull char(40)
+, titrefull_s char(40)
```

Comment:
[Le type le plus approprié est `varchar`](http://stackoverflow.com/q/1827063/2729778), au lieu de `char`. Plus précisément `varchar(512)`, car la taille par défaut (255) n'est pas suffisante tandis qu'à l'heure actuelle 512 fonctionne (`select group_concat(id) from textes_versions group by substr(titrefull, 1, 512) having count(*) > 1 and min(titrefull) <> max(titrefull)` ne retourne rien, contrairement à la même requête avec 255 à la place de 512).